src/documents/RemoteModelAPI.md

```diff
@@ -0,0 +1,364 @@
+---
+title: RemoteModel API
+category: Infusion
+---
+
+`fluid.remoteModelComponent` builds on top of [`fluid.modelComponent`](ComponentGrades.md) with the purpose of providing
+a buffer between a local and remote model that are attempting to stay in sync. For example, a local model is being
+updated by user interaction, this is sent back to a remote server, which in turn attempts to update the local model. If
+additional user actions occur during this roundtrip, an infinite loop of updates may occur. `fluid.remoteModelComponent`
+solves this by restricting reading and writing to a single request at a time, waiting for one request to complete before
+operating the next. Additionally, it will rebase local and remote changes, with the local changes taking priority. (See:
+[Fetch Workflow](#fetch-workflow) for more details on the rebasing.)
+
+## Supported Events
+
+In addition to the standard `fluid.modelComponent` events, `fluid.remoteModelComponent` adds the following:
+
+* `onFetch`,
+* `onFetchError`,
+* `afterFetch`,
+* `onWrite`,
+* `onWriteError`,
+* `afterWrite`
+
+For information on how events work and how to configure listeners to them, see [Infusion Event System](InfusionEventSystem.md).
+
+<div class="infusion-docs-note">
```

Comment:
"This is useful to prevent further actions from happening until all of the listeners have completed" is pretty unclear, as is the next sentence. We should say something like "The transform promise chain pattern is here only being used for sequencing of the effects implemented in the listeners and not to actually transform the payload passed in the listener arguments"